patternfly/patternfly-elements
# chore: pfe-dropdown package.json fix

## <Pull request description>




### Related issues

- (#100) Description of issue
In package.json, main value was set to a wrong value and as a result pfe-dropdown import was giving NOT FOUND error.

### Preview


Link(s) to demo page(s) where this element can be viewed:
- no demo -> package.json setting changed


### What has changed and why



- Changed main value in package.json as it has wrong value and pfe-dropdown couldn't be loaded by using import


### Testing instructions



  1. Use import to load pfe-dropdowm
  2. Add pfe-dropdown element on page

  pfe-dropdowm component should load successfully 


#### Browser requirements

Your component should work in all of the following environments:



### Ready-for-merge Checklist





### Merging

Please **squash** when merging and ensure your commit message uses [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) formatting.

**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**
